js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+xRobotsTagHelp=Verhindern, dass Seiten in Suchmaschinen auftauchen <1>Mehr erfahren</1>
+xXSSProtectionHelp=Dieser Header konfiguriert den Cross-Site-Scripting-Filter (XSS) in Ihrem Browser. Mit dem Standardverhalten verhindert der Browser das Rendern der Seite, wenn ein XSS-Angriff erkannt wird. <1>Mehr erfahren</1>
+strictTransportSecurityHelp=Der HTTP-Header Strict-Transport-Security weist die Browser an, immer HTTPS zu verwenden. Sobald ein Browser diesen Header sieht, wird er die Website nur noch über HTTPS für die angegebene Zeit (1 Jahr) bei max-age besuchen, einschließlich der Subdomains. <1>Mehr erfahren</1>
+bruteForceModeHelpText=Konfiguriert das verhalten der Brute-Force Erkennung.
+refreshTokenMaxReuse=Refresh Token Maximale Wiederverwendung
+refreshTokenMaxReuseHelp=Maximale Anzahl der Wiederverwendung eines Refresh-Tokens. Wenn ein anderes Token verwendet wird, erfolgt der Widerruf sofort.
+policiesConfigType=Konfigurieren mit\:
+profilesConfigTypes.formView=Formular
+attributeGroup=Attributgruppe
+jumpToSection=Bereiche
+generalSettings=Allgemeine Einstellungen
+permission=Berechtigung
+validations=Validierungen
+enabledWhen=Aktiv wenn
+always=Immer
+scopesAsRequested=Wenn Scopes angefordert werden
+enabledWhenTooltip=Gibt an, wann das Attribut verfügbar ist. Wenn Scopes verwendet werden, ist das Attribut verfügbar, wenn mindestens einer der konfigurierten Scopes in der Anmeldeanfrage angewendet wird. Sowohl die Standard- als auch die optionalen Scopes der aktuellen Anfrage werden ausgewertet. In den Kontexten der Adminkonsole und der Benutzerkontokonsole werden die Scopes nicht ausgewertet und das Attribut ist immer verfügbar (wenn es nicht weiter durch Berechtigungen eingeschränkt ist).
+attributeGroupHelp=Gibt die Benutzerprofilgruppe an, der dieses Attribut hinzugefügt wird. Dies ermöglicht die Gruppierung verschiedener ähnlicher Attribute in verschiedenen Teilen des Bildschirms beim Erstellen oder Aktualisieren von Benutzern.
+attributeDisplayNameHelp=Anzeigename für das Attribut. Unterstützt auch Schlüssel für lokalisierte Werte. Zum Beispiel: ${profile.attribute.phoneNumber}. Klicken Sie auf das Symbol "Globe Route", um Übersetzungen hinzuzufügen.
+upAttributeNameHelp=Der Name des Benutzerattributs, der zur eindeutigen Identifizierung eines Attributs dient.
+required=Erforderliches Feld
+requiredHelp=Setzen Sie das Attribut nach Bedarf. Falls aktiviert, muss das Attribut von Benutzern und Administratoren gesetzt werden. Andernfalls ist das Attribut optional.
+requiredFor=Erforderlich für
+requiredForLabel.both=Benutzer und Administratoren
+requiredForLabel.users=Nur Benutzer
+requiredForLabel.admins=Nur Administratoren
+requiredWhen=Erforderlich, wenn
+whoCanEdit=Wer kann bearbeiten?
+whoCanEditHelp=Wenn diese Option aktiviert ist, können Benutzer oder Administratoren das Attribut anzeigen und bearbeiten. Andernfalls haben Benutzer oder Administratoren keinen Zugang zum Schreiben des Attributs.
```

Comment:
"keinen Schreibzugriff auf das Attribut"?

Reply:
Changed to:
```
whoCanEditHelp=... keinen Schreibzugriff auf das Attribut.
```